client/components/checkbox/style.scss

```diff
@@ -0,0 +1,27 @@
+.form-checkbox {
+	margin-right: 8px;
+	cursor: pointer;
+	display: inline-flex;
+	align-items: center;
+	position: relative;
+
+	input {
+		margin: 0;
+
+		&:before {
+			display: none !important; // Prevent Calypso's Noticons font
+		}
+	}
+
+	.gridicon {
+		color: $blue-medium;
+		pointer-events: none;
+		position: absolute;
+		left: 1px;
+		top: 1px;
+	}
+
+	&.is-disabled .gridicon {
+		color: lighten( $gray, 10% );
```

Comment:
Looks like we never have disabled checkboxes? Also, the grey color doesn't work me in Chrome (blue is still showing).